meal_plan_overview/receipt.php

```diff
@@ -0,0 +1,104 @@
+<?php
+include "../shared/php/database.php";
+
+$conn = connect_local();
+header("Content-Type: application/json");
+$request = file_get_contents("php://input");
+
+if (isset($request) && !empty($request)) {
+    $receipts = get_receipt_by_user($conn, json_decode($request)->userId);
+    if (isset($receipts["errorText"])) {
```

Comment:
Rezept is called recipe or recipes

Reply:
I think you need to change that everywhere :)

Reply:
Yes receipt and recipe I mix them always up 😂
I will fix it